src/app/admin-panel/admin-panel/students/students.component.ts

```diff
@@ -66,8 +66,18 @@ export class StudentsComponent implements OnInit {
   }
 
   createStudent() {
-    this.newStudent.classId = this.selectedClassId;
-    // this.newStudent = new Student();
+    this.newStudent = new Student(
```

Comment:
define default values in models constructor

Reply:
We must do this for adding class id.